src/main/java/com/mineboundteam/minebound/event/Events.java

```diff
@@ -0,0 +1,26 @@
+package com.mineboundteam.minebound.event;
+
+
+import com.mineboundteam.minebound.MineBound;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.Items;
+import net.minecraftforge.event.entity.player.PlayerInteractEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+
+@Mod.EventBusSubscriber(modid = MineBound.MOD_ID)
+public class Events {
```

Comment:
Please rename to something more descriptive for organization purposes, such as BreakBlockEvent

Reply:
Done.